main.css

```
@@ -386,17 +365,268 @@ form > textarea:focus {
 
 .footer--section {
   width: 375px;
+  border-top: solid 1px white;
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
-.supporting-item-container:nth-child(2) {
-  margin: 25px 0 15px 0;
-}
-
 .footer--email {
+  font-size: 2rem;
   text-decoration: none;
   color: #fff;
-  margin-bottom: 25px;
+  margin-top: -2.5rem;
+  margin-bottom: 2.5rem;
+}
+
+@media screen and (min-width: 768px) {
```

- [ ] You should use a bigger breakpoint than 768px. Anything below 992px considered as tablet resolution (which has similar UX to mobile version). 
```suggestion
@media screen and (min-width: 992px) {
```